cfgov/core/utils.py

```diff
@@ -65,34 +62,12 @@
 ]
 
 
-def should_interstitial(url: str) -> bool:
-    match = LINK_PATTERN.match(url)
-
-    # If this is another link to .gov do not interstitial.
-    if match.group("domain").endswith(".gov"):
-        return False
-
-    # If this is not a link to a .gov, but it's still subject to CFPB's privacy
-    # policy, do not interstitial
-    elif match.group("domain") in settings.ALLOWED_LINKS_WITHOUT_INTERSTITIAL:
-        return False
-
-    return True
-
-
 def sign_url(url):
     signer = Signer(sep="||")
     url, signature = signer.sign(url).split("||")
     return (url, signature)
```

Comment:
Without the interstitial, `sign_url` should also be unnecessary.

Reply:
https://github.com/cfpb/consumerfinance.gov/pull/8095/commits/b7de2d452e4793722331e7d9af738f95bafd431d